08/kth-missing-positive-number.checkpoint.md

```diff
@@ -208,6 +208,106 @@ Code solution to a subproblem
 
 <!-- ======================= END CHALLENGE ======================= -->
 
+<!-- >>>>>>>>>>>>>>>>>>>>>> BEGIN CHALLENGE >>>>>>>>>>>>>>>>>>>>>> -->
+<!-- Replace everything in square brackets [] and remove brackets  -->
+
+### !challenge
+
+* type: code-snippet
+* language: python3.6
+* id: 5a380194-96e2-4b87-9aa7-2fe407448e1f
+* title: Coding Solution
+* points: 3
+* topics: python, pse, python-lists
+
+##### !question
+
+Write the code to solve the problem here.  You can use the tests to guide your solution.
```

Comment:
What tests? Am I missing how the students would see the tests?

I feel like the first sentence is awkward (Wouldn't we say "Code" as a verb instead of "write the code"? And isn't "the code to solve the problem" both obvious, but also ambiguous, since there isn't really a problem, but tests to pass?) I'm suggesting rewording it. Also, since this won't pass unless the tests pass, the students must use the tests, rather than a suggestive "You can" haha

```suggestion
Implement `kth_missing_positive_number`.  Use these tests to guide your solution.
```

Personally, I think it'd be additionally awesome to acknowledge that these tests will make assumptions that they might not have been made in the earlier questions.

```suggestion
Implement `kth_missing_positive_number`.

Use these tests to guide your solution. These tests may make different assumptions about the problem than you did! Do not alter your assumptions above, and take these assumptions for this challenge.
```